lib/TestDbServer/Command/DeleteTemplate.pm

```diff
@@ -12,6 +12,15 @@ sub _entity_find_method { 'find_template' }
 sub _entity_id_method { 'template_id' }
 sub _not_found_exception { 'Exception::TemplateNotFound' }
 
+sub execute {
+    my $self = shift;
+
+    my $databases_with_this_template = $self->schema->search_database(template_id => $self->template_id);
```

Comment:
At this point it hasn't validated that the `template_id` is valid but that doesn't look like it causes any problems.